STO: 968
VERSION: 969
BRAKETEST_REQUEST: 973
OPERATION_MODE: 974
